app/models/other/gambling/mgd/DoYouHaveMGDRegistrationNo.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.other.gambling.mgd
+
+import utils.{Enumerable, RadioOption, WithName}
+
+sealed trait DoYouHaveMGDRegistrationNo
```

Comment:
Why is this called `DoYouHaveMGDRegistrationNo` when it is for both yes and no values
?